### PR TITLE
New Version: RenewedVision.ProPresenter.7.8.2_117965313

### PR DESCRIPTION
**Pull request by JasonCook599**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/46458)

### CODE REVIEW

File: manifests/r/RenewedVision/ProPresenter/7.8.2+117965313/RenewedVision.ProPresenter.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: RenewedVision.ProPresenter
+PackageVersion: 7.8.2+117965313
```
Comment (ImJoakim) [inline manifests/r/RenewedVision/ProPresenter/7.8.2+117965313/RenewedVision.ProPresenter.installer.yaml:5]:
```suggestion
PackageVersion: 7.8.2_117965313
```

File: manifests/r/RenewedVision/ProPresenter/7.8.2+117965313/RenewedVision.ProPresenter.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: RenewedVision.ProPresenter
+PackageVersion: 7.8.2+117965313
```
Comment (ImJoakim) [inline manifests/r/RenewedVision/ProPresenter/7.8.2+117965313/RenewedVision.ProPresenter.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 7.8.2_117965313
```

File: manifests/r/RenewedVision/ProPresenter/7.8.2+117965313/RenewedVision.ProPresenter.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: RenewedVision.ProPresenter
+PackageVersion: 7.8.2+117965313
```
Comment (ImJoakim) [inline manifests/r/RenewedVision/ProPresenter/7.8.2+117965313/RenewedVision.ProPresenter.yaml:5]:
```suggestion
PackageVersion: 7.8.2_117965313
```

**Review by ImJoakim — CHANGES_REQUESTED** [review]
image

You would also need to update the version folder as well:

`RenewedVision/ProPresenter/7.8.2+117965313/` -> `RenewedVision/ProPresenter/7.8.2_117965313/`

**Review by ImJoakim — APPROVED** [review]

**Comment by JasonCook599** [comment]
> You would also need to update the version folder as well:
> 
> `RenewedVision/ProPresenter/7.8.2+117965313/` -> `RenewedVision/ProPresenter/7.8.2_117965313/`

Just wondering why an underscore is recommended here. Semantic versioning expects build metadata after a plus sign. Is there any documentation you can point me to? I searched but couldn't find anything relevant.

**Pull request merged**